foreign_cc/private/cmake_script.bzl

```diff
@@ -31,6 +31,9 @@ _TARGET_ARCH_PARAMS = {
     "aarch64": {
         "CMAKE_SYSTEM_PROCESSOR": "aarch64",
     },
+    "s390x": {
```

Comment:
There's probably a lot more values that are missing from here as `CMAKE_SYSTEM_PROCESSOR` can take many values (see https://stackoverflow.com/questions/70475665/what-are-the-possible-values-of-cmake-system-processor). At a minimum PowerPC is missing, although its not that common and is even less comonly used by bazel users IME. I guess we can let people add additional cases as they come across them though..

Reply:
I was going to add ppc64le to the list (that's a platform I can actually test on) but ppc64le was added to `@platforms` late, and it's not actually available in the version of `platforms` currently loaded by the tests. That's fixable with a bump of `@platforms`, though, so I'd be glad to do that if you want.

Reply:
Okay, I elected not to do this because it requires that all consuming projects update their platforms version. (I'd be happy to do that if you think it's important, but I realized it's a breaking change)

Reply:
I've not come across anyone actively adding any fixes for ppc64le so lets assume for now that there is no one using this to avoid the breaking change (although we're still pre 1.0.0 so technically there is no stability guarantee between versions)